les/ulc_test.go

```diff
@@ -40,6 +40,48 @@ func TestULCSyncWithOnePeer(t *testing.T) {
 	}
 }
 
+func TestULCSyncWithOnePeer1(t *testing.T) {
+	f := newFullPeerPair(t, 1, 4, testChainGen)
+	ulcConfig := &eth.ULCConfig{
+		MinTrustedFraction: 100,
+		TrustedNodes:       []string{f.ID.String()},
+	}
+
+	l := newLightPeer(t, ulcConfig)
+
+	if reflect.DeepEqual(f.PM.blockchain.CurrentHeader().Hash(), l.PM.blockchain.CurrentHeader().Hash()) {
```

Comment:
Why this check? Is there any chance that these new and different instances share the same header? If yes I also would use a more detailed message, e.g. "header blocks of full and light peer are equal".